up42/storage.py

```diff
@@ -362,6 +367,11 @@ def get_orders(
             )
         sort = f"{sortby},{'desc' if descending else 'asc'}"
         url = f"{self.auth._endpoint()}/workspaces/{self.workspace_id}/orders?format=paginated&sort={sort}"
+        if order_type is not None and order_type in ["TASKING", "ARCHIVE"]:
```

Comment:
maybe we should raise a warning that order type will be ignored if not in `["TASKING", "ARCHIVE"]`.
Something similar to what you did [here](https://github.com/up42/up42-py/blob/dfaeb2002f11dc2533b9c30bc26337c8684fc6ca/up42/tasking.py#L164).

Reply:
✅